### PR TITLE
Blog post for scala 2.10 and java 7

### PR DESCRIPTION
**Pull request by kevinoliver**
Notify our users of the plan to do a final set of Scala 2.10/Java 7 releases.

### CODE REVIEW

File: source/blog/2016-04-20-scala-210-and-java7.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer, Ostrich
+---
+
+The upcoming releases of our family of projects will be the final release for
+Scala 2.10 and Java 7. These should be Finagle 6.35.0, Finatra 2.2.0,
+Util 6.34.0, Scrooge 4.7.0, TwitterServer 1.20.0, and Ostrich 9.18.0.
```
Comment (vkostyukov) [inline source/blog/2016-04-20-scala-210-and-java7.md]: Should we mention that we're still going to publish 2.10 jars for Scrooge (right?) since scrooge-sbt-plugin needs that.

File: source/blog/2016-04-20-scala-210-and-java7.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+title: Final Scala 2.10 and Java 7 release
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer, Ostrich
+---
+
+The upcoming releases of our family of projects will be the final release for
+Scala 2.10 and Java 7. These should be Finagle 6.35.0, Finatra 2.2.0,
```
Comment (taylorleese) [inline source/blog/2016-04-20-scala-210-and-java7.md]: FFTI but I would bullet this list.

File: source/blog/2016-04-20-scala-210-and-java7.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Util 6.34.0, Scrooge 4.7.0, TwitterServer 1.20.0, and Ostrich 9.18.0.
+
+The reason for this is because Java 7 hit it's
```
Comment (taylorleese) [inline source/blog/2016-04-20-scala-210-and-java7.md]: Suggested verbiage change:
Java 7 reached it's [end-of-life](http://www.oracle.com/technetwork/java/eol-135779.html) in April 2015 and we would like to begin using the Java 8 libraries as well as remove some Java 7 specific workarounds.

File: source/blog/2016-04-20-scala-210-and-java7.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+[end-of-life](http://www.oracle.com/technetwork/java/eol-135779.html)
+back in April 2015. We would like to begin using the Java 8 libraries in
+our code and remove some Java 7 specific workarounds. Once Scala 2.12 is
+released we will work on adding 2.11/2.12 cross-compile support.
```
Comment (taylorleese) [inline source/blog/2016-04-20-scala-210-and-java7.md]: Suggested verbiage change:
Once Scala 2.12 is released, we plan to add support for cross-compiling Scala 2.11 and 2.12.

Comment (kevinoliver) [inline source/blog/2016-04-20-scala-210-and-java7.md]: taken.

File: source/blog/2016-04-20-scala-210-and-java7.md (hunk 3 above)
Comment (kevinoliver) [inline source/blog/2016-04-20-scala-210-and-java7.md]: taken.

File: source/blog/2016-04-20-scala-210-and-java7.md (hunk 2 above)
Comment (kevinoliver) [inline source/blog/2016-04-20-scala-210-and-java7.md]: bulleted.

File: source/blog/2016-04-20-scala-210-and-java7.md (hunk 1 above)
Comment (kevinoliver) [inline source/blog/2016-04-20-scala-210-and-java7.md]: yep.

**Comment by kevinoliver** [comment]
@taylorleese @vkostyukov rebased and pushed a new version.

**Comment by mosesn** [comment]
LGTM

File: source/blog/2016-04-20-scala-210-and-java7.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+remove some Java 7 specific workarounds. Once Scala 2.12 is released, we plan
+to add support for cross-compiling Scala 2.11 and 2.12.
+
+Note that the scrooge-sbt-plugin will continue to be published for only
+Scala 2.10 due to sbt plugin requirements.
+
+If important patch releases become necessary we will try to accomodate these
+needs.
+
+Please let us know if you have any questions, either by
```
Comment (cacoco) [inline source/blog/2016-04-20-scala-210-and-java7.md]: Can you also add finatra? 

[@finatra](https://twitter.com/finatra) and [Finatra mailing list](https://groups.google.com/forum/#!forum/finatra-users), or [finatra chat](https://gitter.im/twitter/finatra)

File: source/blog/2016-04-20-scala-210-and-java7.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer, Ostrich
+---
+
+The upcoming releases of our family of projects will be the final release for
+Scala 2.10 and Java 7. These should be:
+
+* Finagle 6.35.0
+* Finatra 2.2.0
```
Comment (cacoco) [inline source/blog/2016-04-20-scala-210-and-java7.md]: Finatra 2.1.6 is the next version.

Comment (kevinoliver) [inline source/blog/2016-04-20-scala-210-and-java7.md]: i'll change it, but i'm surprised you're not gonna bump minor even with all the dependency changes.

File: source/blog/2016-04-20-scala-210-and-java7.md (hunk 5 above)
Comment (kevinoliver) [inline source/blog/2016-04-20-scala-210-and-java7.md]: yep.

**Comment by kevinoliver** [comment]
@cacoco new version rebased and pushed with your feedback

File: source/blog/2016-04-20-scala-210-and-java7.md (hunk 6 above)
Comment (cacoco) [inline source/blog/2016-04-20-scala-210-and-java7.md]: I think our  2.11 only release would be the 2.2.0 release as we'd keep continuity in our 2.1.x releases being Scala 2.10 compatible. But maybe that doesn't make sense?

**Comment by cacoco** [comment]
Looks great, thanks!

File: source/blog/2016-04-20-scala-210-and-java7.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+Scala 2.10 and Java 7. These should be:
+
+* Finagle 6.35.0
+* Finatra 2.1.6
+* Util 6.34.0
+* Scrooge 4.7.0
+* TwitterServer 1.20.0
+* Ostrich 9.18.0
+
+Java 7 reached it's [end-of-life](http://www.oracle.com/technetwork/java/eol-135779.html)
```
Comment (travisbrown) [inline source/blog/2016-04-20-scala-210-and-java7.md]: Sorry about the drive-by nitpick but s/it's/its/.

Comment (kevinoliver) [inline source/blog/2016-04-20-scala-210-and-java7.md]: hi @travisbrown, thank's :)

(fixed)
